### PR TITLE
[Python PQ/RQ] Meet Python

### PR DESCRIPTION
**Pull request by sebaraba**
## Done:

- [x] python-2 PQ
- [x] python-3 PQ
- [x] python-hello-world PQ
- [x] why-learn-python PQ

### CODE REVIEW

File: python/core/meet-python/python-2.md [hunk 1]
```diff
@@ -51,6 +52,18 @@ Python 3 is considered to be an improved version of Python 2, with fewer mistake
 
 As a beginner, your best bet is **Python 3**. This is the standard that's advancing slightly faster and you don't have to worry at all about porting a whole Python 2 application to Python 3 (the main reason why Python 2 hung around for so long).
 
+---
+## Practice
+
+To print to console in python 3 we use a:
+
```
Comment (lizTheDeveloper) [inline python/core/meet-python/python-2.md]: rewrite this question, doesn't really discuss what the article covers -
what objective is this intended to cover?

File: python/core/meet-python/python-3.md [hunk 2]
```diff
@@ -40,14 +40,25 @@ $
...
 To run Python file, it should firstly have the `.py` extension. Then, invoke a new terminal inside the directory and write:
 ```bash
 $ python3 myNewPython.py
 ```
 The process couldn't be simpler!
 
+---
+## Practice
+
+What is python shell called like and what does the acronym stand for?
```
Comment (lizTheDeveloper) [inline python/core/meet-python/python-3.md]: When you run  `python3`, this is called ???
and then add more plausible answers

File: python/core/meet-python/python-hello-world.md [hunk 3]
```diff
@@ -43,10 +43,24 @@ This produces the following output:
 # Hello World!
 ```
 
+---
+
+## Practice
+
+Complete the following code snippet such that it respects python3 syntax:
```
Comment (lizTheDeveloper) [inline python/core/meet-python/python-hello-world.md]: Update the following code snippet to be compatible with Python 3

File: python/core/meet-python/why-learn-python.md [hunk 4]
```diff
@@ -32,7 +33,17 @@ for i in items:
 
 Python is also an economical programming language. For similar tasks, Python can often require *3-5* times less code than Java, and up to *10* times less code than C++.
 
-Python is an *object-oriented* programming language, therefore it can act as a stepping stone to learning new object-oriented languages, and can help you adapt to other programming environments.
+As we know, Python is an *object oriented* programming language, therefore because of Python's object oriented nature it can act as a stepping stone to learning new object oriented languages, and can help you adapt to other programming environments. Nonetheless, Python is also used as functional programming language in certain conditions. 
+
+---
+## Practice
+
+What is the *pythonic* way to write a for loop?
```
Comment (lizTheDeveloper) [inline python/core/meet-python/why-learn-python.md]: Expand the code sample to include an actual loop
How does this discuss what's covered in the article?

Comment (sebaraba) [reply python/core/meet-python/why-learn-python.md]: @lizTheDeveloper please review this question again, I tried to change it such that it reflects the objectives of this insight, not sure whether it is accurate tho.

**Pull request closed without merging**